index.js

```diff
@@ -4,7 +4,39 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    let result =0, 
+        operator = process.argv[2], // Getting operator
+        numberArray = (process.argv.splice(3, process.argv.length)).map(Number); // Getting num array as string || creating new number Array
```

Comment:
style comment: better to write three separate variable declatations, each using let or const.